modules/ldap-syncing-config-rfc2307.adoc

```diff
@@ -52,12 +55,13 @@ stored.
 <2> When `false`, secure
 LDAP (`ldaps://`) URLs connect using TLS, and insecure LDAP (`ldap://`) URLs are
 upgraded to TLS. When `true`, no TLS connection is made to the server and you cannot use `ldaps://` URL schemes.
-<3> The attribute that uniquely identifies a group on the LDAP server.
+<3> Specify the `bindDN`.
```

Comment:
@stlaz actually no, it should not be left out. Because, usually the admin skips that part and goes directly to define the ldap sync config and the docs imply that you don't need bindDN and bindPassword. In any config, the full required options must be added.

Reply:
@stlaz I know you said it might not be necessary because it's covered elsewhere, but can we go ahead with this change to help make it clearer that it's needed?

Reply:
In that case, if you want to outline `bindDN`, you should explain it better than just "specify the bindDN" as that's not very helpful. Or just leave out the explanation completely.

Reply:
@stlaz Good point. I've removed the description since what was there wasn't adding any value. 

As for your other question about bindPassword file, that is taken exactly from another example. Are you saying that this won't work / needs changed somehow? Or are we okay with what we have